### PR TITLE
pkp/pkp-lib#9823 update the crontrol job processing via scheduler

### PR DESCRIPTION
**Pull request by touhidurabir**
for pkp/pkp-lib#9823

### CODE REVIEW

File: docs/release-notes/README-3.5.0 [hunk 1]
```diff
@@ -24,8 +24,12 @@ New config.inc.php section for schedule task named schedule is added, with the f
 
 - The setting general.scheduled_tasks has been removed. It was supposed to control the schedule tasks but had no impact.
 
+New config.inc.php parameters added for queues:
+    - schedule_job_process (default value: On), control the process of queue jobs via task scheduler
```
Comment (jonasraoni) [inline docs/release-notes/README-3.5.0:28]:
```suggestion
    - schedule_job_process (default value: Off): Controls whether queued jobs should be processed by the task scheduler.
```

**Review by jonasraoni — CHANGES_REQUESTED** [review]
@touhidurabir Just minor changes, I think the setting name is difficult for me to understand, so I've suggested alternatives.

File: config.TEMPLATE.inc.php [hunk 2]
```diff
@@ -566,6 +566,11 @@
 ; memory_limit the server has configured for PHP.
 job_runner_max_memory = 80
 
+; Allow queue jobs to be processed via schedule task.
+; Note that when scheduler running in web request mode and if the job_runner is enabled
+; it will not process jobs via schedule task as both approach use same mechanism.
+; schedule_job_process = On
```
[the remark this answers is not in the document]
Comment (touhidurabir) [reply config.TEMPLATE.inc.php:572]: @jonasraoni 
> Another alternative would be to move this setting to the [schedule] group as process_queued_jobs = On|Off

I think it's better to keep in within `queue` section as it's a queue jobs execution control option .

> suggest changing the name

I agree the current name is bit difficult to understand and the purpose it serve. My motivation was to keep the name bit small but I also think it makes it bit harder to understand. I agree with changing the name as you proposed .

[the remark this answers is not in the document]
Comment (jonasraoni) [reply config.TEMPLATE.inc.php]: Ok! I'll approve everything 😁

**Review by jonasraoni — APPROVED** [review]

**Pull request merged**